projects/control-service/cicd/.gitlab-ci.yml

```diff
@@ -4,14 +4,30 @@
     when:
       - always
 
+
+# This grouping represents ALL files related to the control plane.
+# This grouping triggers actions like new releases etc. but don't trigger unit or integration tests.before_script
+# because it also includes the helm chart folders and we don't want to run unit and integration tests anytime
+# the helm chart changes
 .control_service_changes: &control_service_change_locations
   - projects/control-service/cicd/**/*
   - projects/control-service/projects/base/**/*
   - projects/control-service/projects/model/**/*
   - projects/control-service/projects/pipelines_control_service/**/*
   - projects/control-service/projects/settings.gradle
+  - projects/control-service/projects/versions-of-external-dependencies.gradle
   - projects/control-service/projects/helm_charts/pipelines-control-service/**/*
 
+# This is a subset of the &control_service_change_locations which listens on changes to all code
+# Changes to this group should trigger runs of runs of unit and integration tests.
+.control_service_code_changes: &control_service_code_change_locations
```

Comment:
Our helm chart deployment uses the current commit as tag for the image of the control service. See [here](https://github.com/vmware/versatile-data-kit/blob/main/projects/control-service/cicd/deploy-testing-pipelines-service.sh#L16)

Reply:
Yeah that is fine. 
That script is triggered in a section listening on `&control_service_change_locations`.
What do you see as being the issue?